coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -9,6 +13,10 @@ class DocumentationComment:
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
+    _ParseMode = enum("DESCRIPTION", "PARAM", "RETVAL")
+    Param = namedtuple('Param', 'name, desc')
+    RetVal = namedtuple('RetVal', 'desc')
+    Desc = namedtuple('Desc', 'desc')
```

Comment:
and why not calling them `Parameter`, `ReturnValue` and `Description`? slightly more verbose but I think it's okay^^